### PR TITLE
ORCA-96: Domo Integration

### PR DESCRIPTION
**Pull request by secretsayan**

### CODE REVIEW

**Review by TravisCarden — CHANGES_REQUESTED** [review]
Just a few changes.

File: src/Domain/Composer/Version/DrupalCoreVersionResolver.php [hunk 1]
```diff
@@ -426,6 +426,9 @@ private function assertNextMajorLatestMinorBetaOrLaterExists(): void {
     $is_alpha = strpos($stability, "alpha");
     $is_dev = strpos($stability, "dev");
     if ($is_alpha || $is_dev) {
+      // Assigning to null as beta or later version does not exist.
+      $this->nextMajorLatestMinorBetaOrLater = NULL;
```
[the remark this answers is not in the document]
Comment (secretsayan) [reply src/Domain/Composer/Version/DrupalCoreVersionResolver.php:430]: Please suggest a function name for the same :)

**Review by TravisCarden — APPROVED** [review]

**Comment by danepowell** [comment]
@secretsayan @TravisCarden it looks like the ORCA workflow example integration was never updated to add these new secrets. Was that intentional? https://github.com/acquia/orca/blob/develop/example/.github/workflows/orca.yml

**Comment by secretsayan** [comment]
@danepowell Thanks for pointing this out. PR https://github.com/acquia/orca/pull/425 created to modify example files.

**Pull request merged**